db/migrate/20191115103058_remove_document_type_id_from_documents.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class RemoveDocumentTypeIdFromDocuments < ActiveRecord::Migration[5.2]
+  def up
```

Comment:
The `up`/`down` methods could be replaced by a single `change` method, e.g.

```
def change
  remove_column :documents, :document_type_id, :string
end
```

Since the column type is defined in the `remove_column`, the migration is reversible.

Reply:
ah nice, hadn't realised that. Will update, thanks :)